migrations/20160625024216-adding_rating_table.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+module.exports = {
+    up: function (queryInterface, Sequelize) {
+        /*
+            Add altering commands here.
+            Return a promise to correctly handle asynchronicity.
+
+            Example:
+            return queryInterface.createTable('users', { id: Sequelize.INTEGER });
+        */
+        return queryInterface.createTable('LichessRatings', {
```

Comment:
The idea behind an ORM like this is to be able to easily switch between DB types. Of course, it's never perfectly replaceable in practice, but for projects like these - it's close.